subsys/bluetooth/controller/hci_internal.h

```diff
@@ -24,6 +27,32 @@
  */
 int hci_internal_cmd_put(uint8_t *cmd_in);
 
+/** A user implementable HCI command handler
+ *
+ * @param[in]  cmd The HCI command itself
```

Comment:
What do you mean by "The HCI command itself"?

Reply:
The HCI command buffer as formatted defined by the spec.

Maybe add
`The first byte in the buffer should correspond to OpCode, as specified by the Bluetooth Core Specification.` ?